### ISSUE TITLE
wbomb6 issues

### ISSUE THREAD

**Issue by Triang3l** [Qfusion/qfusion#79]
There are some bugs on wbomb6 "Depot".

First, one spawn point has its cross decal displayed incorrectly:
![wsw_140428_203257](https://cloud.githubusercontent.com/assets/1764343/2820415/d063b86a-cef9-11e3-993e-0c5f58d7d336.jpg)

I think it would be good to move it to the left side of the box, where it won't be blocked by campers and won't be in the middle of battle:
![wsw_140428_212532](https://cloud.githubusercontent.com/assets/1764343/2820436/16a3ea84-cefa-11e3-9afd-7d28fd9d7b28.jpg)

So, `"origin" "616.000000 2800.000000 240.000000"` should be changed to `704.000000 2800.000000 136.000000`.

Another issue is the lack of clip brushes on certain pitched roofs. One example is at `-1080.1250 2900.3125 759.9375 0 10`:
![wsw_140428_213052](https://cloud.githubusercontent.com/assets/1764343/2820493/d4921c96-cefa-11e3-80dd-5e77d59b234a.jpg)

**Comment by viciious** [comment]
Leaving this for reference: http://www.warsow.net/forum/thread/t/224147#post-224147
